cumulusci/cli/cci.py

```diff
@@ -624,31 +614,40 @@ def project_dependencies(config):
         click.echo(line)
 
 
-project.add_command(project_init)
-project.add_command(project_info)
-project.add_command(project_dependencies)
-
-
 # Commands for group: service
 
 
-@click.command(name="list", help="List services available for configuration and use")
+@service.command(name="list", help="List services available for configuration and use")
+@click.option("--plain", is_flag=True, help="Print the table using plain ascii.")
+@click.option("--json", "print_json", is_flag=True, help="Print a json string")
 @pass_config(allow_global_keychain=True)
-def service_list(config):
-    headers = ["service", "description", "is_configured"]
-    data = []
+def service_list(config, plain, print_json):
     services = (
         config.project_config.services
         if not config.is_global_keychain
         else config.global_config.services
     )
+    configured_services = config.keychain.list_services()
+    plain = plain or config.global_config.cli_options__plain_output
+
+    data = [["Name", "Description", "Configured"]]
     for serv, schema in services.items():
-        is_configured = ""
-        if serv in config.keychain.list_services():
-            is_configured = "* "
-        data.append((serv, schema["description"], is_configured))
-    table = Table(data, headers)
-    click.echo(table)
+        is_configured = serv in configured_services
+        data.append([serv, schema["description"], is_configured])
+
+    if print_json:
+        click.echo(json.dumps(data[1:]))
+        return None
```

Comment:
Yeah, that was the consensus from the demo as well. For a first pass it made sense to use the list-of-lists structure since it was shared with the previous library, but I think a `csv.DictWriter`-style API that can accept a list-of-dicts makes much more sense. We can then serialize those in a more useful way.

I'll back these out for the initial implementation.